src/SciAssist/pipelines/__init__.py

```diff
@@ -59,7 +63,7 @@
 
 }
 
-def load_model(config: Dict, cache_dir=BASE_CACHE_DIR, device="gpu"):
+def load_model(config: Dict, checkpoint=None, cache_dir=BASE_CACHE_DIR, device="gpu"):
```

Comment:
This variable _checkpoint_ is used to enable user-customized checkpoint path, right?

Reply:
> This variable _checkpoint_ is used to enable user-customized checkpoint path, right?

It works in the same way with huggingface. Users can choose a local path or a repo name from Hub. Checkpoint is used to initialize the model.